olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/Types.h

```diff
@@ -78,6 +82,12 @@ using PrefetchTilesResponse = Response<PrefetchTilesResult>;
 /// Prefetch completion callback type
 using PrefetchTilesResponseCallback = Callback<PrefetchTilesResult>;
 
+/// Subscribe response type
+using SubscribeApiResponse = Response<model::SubscribeResponse>;
+
+/// Delete subscription response type
+using DeleteSubscriptionApiResponse = Response<int>;
```

Comment:
I wanted to link the subscribe and delete subscription namely, so i named them SubscribeResponse and UnsubscribeResponse.